aries_cloudagent/anoncreds/routes.py

```diff
@@ -307,28 +307,27 @@ class CredDefPostOptionsSchema(OpenAPISchema):
         metadata={
             "description": endorser_connection_id_description,
             "example": UUIDFour.EXAMPLE,
-            "required": False,
-        }
+        },
+        required=False,
     )
     create_transaction_for_endorser = fields.Bool(
         metadata={
             "description": create_transaction_for_endorser_description,
             "example": False,
-            "required": False,
-        }
+        },
+        required=False,
     )
     support_revocation = fields.Bool(
         metadata={
             "description": "Support credential revocation",
-            "required": False,
-        }
+        },
+        required=False,
     )
     revocation_registry_size = fields.Int(
         metadata={
             "description": "Maximum number of credential revocations per registry",
-            "example": 666,
-            "required": False,
-        }
+        },
+        required=False,
```

Comment:
In the messaging/credential_definition/routes.py, we have:
```py
    revocation_registry_size = fields.Int(
        required=False,
        validate=INDY_REV_REG_SIZE_VALIDATE,
        metadata={
            "description": "Revocation registry size",
            "strict": True,
            "example": INDY_REV_REG_SIZE_EXAMPLE,
        },
    )
```
where validation is included that the revocation registry size must be >= 4 and <= 32768. Will it make sense to include that range validation to this field as well?

Reply:
@jamshale it seems reasonable to expect the validation to be the same for this field, right?

Reply:
Yes. That should be here. Can add it in another upcoming anoncreds PR to not block this.

Reply:
Thanks @jamshale !